README.md

```diff
@@ -9,14 +9,53 @@ which incidents. Its deployment happens once a day via
 
 ## Getting Started
 
-  To get started all you need is an empty PostgreSQL database and the following dependencies:
+### Install Dependencies
 
-    $ sudo zypper in -C postgresql-server postgresql-contrib
-    $ sudo zypper in -C perl-Mojolicious perl-Mojolicious-Plugin-Webpack \
-      perl-Mojo-Pg perl-Cpanel-JSON-XS perl-JSON-Validator perl-IO-Socket-SSL nodejs16
-    $ npm install
+To install all required dependencies, run
 
-Update the config file `dashboard.yml` to point to your PostgreSQL database (and other services where appropriate):
+    sudo zypper in -C postgresql-server postgresql-contrib
+    sudo zypper in -C perl-Mojolicious perl-Mojolicious-Plugin-Webpack \
+      perl-Mojo-Pg perl-Cpanel-JSON-XS perl-JSON-Validator perl-IO-Socket-SSL nodejs18
```

Comment:
We don't need to install an old version of nodejs. The current/default version works fine. I'd explicitly mention npm, though:
```suggestion
            perl-Mojo-Pg perl-Cpanel-JSON-XS perl-JSON-Validator perl-IO-Socket-SSL nodejs npm
```

Reply:
I've added `npm`, however as there is no `nodejs` package, i added `nodejs22`, alternatively we could use`nodejs-default`.

Reply:
I would prefer to use `nodejs-default` - unless that points to a very old version.

Reply:
If i remember correctly the reason i originally set it to nodejs16 was that it was the newest version back then, and we needed some features not available in older releases. Any newer version should be fine now.